packages/experimental/RadioGroup/src/RadioGroup/RadioGroupTokens.ts

```diff
@@ -1,9 +1,16 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { RadioGroupTokens } from './RadioGroup.types';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
 
 export const defaultRadioGroupTokens: TokenSettings<RadioGroupTokens, Theme> = (t: Theme) =>
   ({
     color: t.colors.menuItemText,
     variant: 'subtitle2Strong',
+    disabled: {
+      color: t.colors.neutralForegroundDisabled,
+      fontFamily: globalTokens.font.family.default,
```

Comment:
Are these the same fontFamily/fontSize/fontWEight as the variant specified above? If so, you shouldn't need to add these

Reply:
Yes, they're the same. Removed!